include/ktopt_interface/ktopt_planning_context.hpp

```diff
@@ -42,6 +47,14 @@ class KTOptPlanningContext : public planning_interface::PlanningContext
     void clear() override;
 
     void setRobotDescription(std::string robot_description);
+    VectorXd moveit_to_drake_complete_state(
```

Comment:
Make sure you're consistent with the naming, functions should be camelCase, so please rename these to moveitToDrakeCompleteState or somehing similar. Maybe call it getXYFromRobotState?